src/Nethermind/Nethermind.Network.Test/P2P/Subprotocols/Eth/V62/TxFloodControllerTests.cs

```diff
@@ -46,56 +49,58 @@ public void Setup()
                 Substitute.For<ITxPool>(),
                 LimboLogs.Instance);
 
-            _controller = new TxFloodController(_handler, LimboNoErrorLogger.Instance);
+            _timestamper = Substitute.For<ITimestamper>();
+            _timestamper.UtcNow.Returns(c => DateTime.UtcNow);
```

Comment:
there is also a ManualTimestamper for testing (just in case you find it useful in the future)

Reply:
good to know, but makes little difference here